runtime/doc/lua.txt

```diff
@@ -2937,6 +2930,34 @@ vim.fs.find({names}, {opts})                                   *vim.fs.find()*
         (`string[]`) Normalized paths |vim.fs.normalize()| of all matching
         items
 
+vim.fs.findroot({bufnr}, {marker})                         *vim.fs.findroot()*
```

Comment:
Should we name it just root()? That aligns with `vim.fs.dir`, `vim.fs.parents`, etc.

```suggestion
vim.fs.root({bufnr}, {marker})                         *vim.fs.findroot()*
```

Reply:
f7721be5b8178f4f40c633da9c61e4cdf9489a97